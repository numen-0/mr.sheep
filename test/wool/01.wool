main:
    JMP B
; ...
; ...
; ...
A:
    print_c 'A'
    EXIT 0
B:
    print_c 'B'
    EXIT 0
