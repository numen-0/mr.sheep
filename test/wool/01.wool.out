mr.sheep: successfully loaded './test/wool/01.baa' of 10B
B
mr.sheep: program exited with code: 0
